packages/SystemUI/res-keyguard/values-ru/havoc_strings.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/**
+ * Copyright (C) 2019 Havoc-OS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<resources>
+
+    <string name="custom_text_clock_top_text_default">Сейчас</string>
+
+    <string name="text_clock_zero_h_min">Ноль</string>
+    <string name="text_clock_high">Вечер</string>
+    <string name="text_clock_noon">Утро</string>
+
+    <!-- TensString -->
+    <string name="text_clock_tens_string_twenty">Двадцать</string>
+    <string name="text_clock_tens_string_thirty">Тридцать</string>
+    <string name="text_clock_tens_string_forty">Сорок</string>
+    <string name="text_clock_tens_string_fifty">Пятьдесят</string>
+    <string name="text_clock_tens_string_sixty">Шестьдесят</string>
+
+    <!-- UnitsString -->
+    <string name="text_clock_units_string_clock">Часы</string>

```

Comment:
I think here it should be something like `zero` value.

but it's hard to image how it works with code:
https://github.com/Havoc-OS/android_frameworks_base/blob/76d9e1e290725bebec9a7f045cf9f76d6fc6e807/packages/SystemUI/src/com/android/keyguard/clocks/CustomTextClock.java#L251

if it's `3:00` it will be `three clock`?